tests/test_profile_merge.py

```diff
@@ -0,0 +1,30 @@
+#   ########################################################################
+#   Copyright 2021 Splunk Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#   ########################################################################
+
+from unittest import TestCase
+
+from splunk_connect_for_snmp_mib_server.profiles import merge_profiles
+
+
+class ProfileLoaderTest(TestCase):
+    def test_multiple_files_merging(self):
```

Comment:
I would add a test with 2 same profile names to check overwriting, test with incorrectly formatted yaml and test with single file in profiles folder